### DISCUSSION TITLE
Алиасы - писать латиницей или не делать вообще?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#262] [General]
Sushi теперь в алиасах поддерживает только латиницу, мотивировка тут: https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/Multilingual.20FSH.20Online.3F/near/429403573

Есть следующие варианты:
- попробовать убедить рабочую группу FHIR вернуть поддержку национальных языков
- перейти на англоязычные алиасы
- отказаться от алиасов

@Vrodvi Виталий, что думаете?

**Comment by Vrodvi** [comment]
Что-то из этого:
- попробовать убедить рабочую группу FHIR вернуть поддержку национальных языков
- отказаться от алиасов

На мой взгляд точно не вариант "перейти на англоязычные алиасы", т.к. цель использования алиасов была как раз в том чтобы ссылаться на артефакты по нормальным читаемым именам вместо формальных машинных англоязычных имён.
Иначе слишком много слоёв, которые будут искажать смысл исходного значения - исходное наименование артефакта на русском языке будет названо по английски, а затем ещё и укорочено до алиаса - это явно не добавит повышения читаемости профилей.

**Reply by eugenekogan-spb** [reply]
Попробовал, написал следующее.

In a national IG, you often refer to a local code system/naming system, which has a well-known specific name in the national language. In this case, we create an alias for that name to make the reference clearly recognizable. If using local names in aliases for our national IG profiles is not possible, the value of aliases is lost. Is it possible to bring the discussion back to the Working Group?

Пришёл ответ.

Chris Moesel
[12:31 AM](https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/Multilingual.20FSH.20Online.3F/near/434241838)
Hi @Evgenii Kogan. Thanks for raising this concern. At this point, the FSH 3.0 spec is too far in the process to be able to back it up and make any changes. This is something that we will need to discuss for a future version of FSH instead.
The intended value of aliases was to provide short and memorable handles for long complicated URLs -- so that value is not lost. That said, your suggestion of using aliases so that the local well-known names can be used definitely makes sense, and I'm sorry if we inadvertently lost that in the process of this change.
I'll note, however, that although the spec says that only ASCII characters are allowed for alias names, SUSHI logs it as a warning (not an error) and continues processing without failure. So... you can choose to continue using local alias names with non-ASCII characters if you wish, and you can even have SUSHI suppress the warnings via a sushi-ignoreWarnings.txt file (documented [here](https://fshschool.org/docs/sushi/project/#using-the-hl7-ig-publisher-and-auto-builder)):
sushi-ignoreWarnings.txt: This optional file can be used to suppress warnings logged by SUSHI. Errors and informational logs from SUSHI cannot be ignored. This file should be placed either at the root of the project (e.g., my-project/sushi-ignoreWarnings.txt in the example above), or within the input directory (e.g., my-project/input/sushi-ignoreWarnings.txt). Warnings will be ignored if they completely match the contents of any line of the file (one line per warning, case-sensitive). Regular expressions can also be specified, one per line, indicated by starting and ending the line with /.
Just be aware that if you do this, you are technically violating the FSH 3.0 spec -- but that's probably not a problem unless you are using a FSH compiler other than SUSHI (and I'm not aware of any FSH compilers other than SUSHI).

Таким образом, видимо, следует оставить русскоязычные алиасы, игнорируя предупреждения технически или визуально. Так?
@Vrodvi

**Reply by Vrodvi** [reply]
Да, предлагают оставить русскоязычные алиасы и игнорировать предупреждения. 
Заодно потренируемся "гасить" предупреждения через указанный файл настройки - это тоже будет полезно.
